### PR TITLE
Add at a glance attendee information.

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/67176

This PR adds a non sortable (due to missing criteria) column the list table of any ticketing supporting post.

Note: a post can have 0+ tickets, each ticket can have a stock or have an unlimited stock; if just one ticket for a post has unlimited stock then the percentage will not render.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Tickets.php:55](https://github.com/moderntribe/event-tickets/blob/2bc685f8314e3f39a1dc5b863131471b86791ac1/src/Tribe/Admin/Columns/Tickets.php#L55)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Arguments with default values must be at the end of the argument list | [Tickets.php:105](https://github.com/moderntribe/event-tickets/blob/ef90c6cb40de933291835939e8c735c0906ce011/src/Tribe/Admin/Columns/Tickets.php#L105)
:x: | Expected 1 space after &quot;==&quot;; 0 found | [Tickets.php:109](https://github.com/moderntribe/event-tickets/blob/ef90c6cb40de933291835939e8c735c0906ce011/src/Tribe/Admin/Columns/Tickets.php#L109)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Admin/Columns.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		$this->add_custom_columns( (array) tribe_get_option( 'ticket-enabled-post-types', array() ) );
+	}
+
+	protected function add_custom_columns( $supported_types ) {
+		if ( empty( $supported_types ) ) {
+			return true;
+		}
+
+		foreach ( $supported_types as $supported_type ) {
+			$tickets_column = new Tribe__Tickets__Admin__Columns__Tickets( $supported_type );
```
Comment (lucatume) [inline src/Tribe/Admin/Columns.php]: Good idea! I've pushed code to do it.

**Comment by barryhughes** [comment]
:leaves:

**Pull request merged**